src/mongo/installer/msi/wxs/UIFragment.wxs

```diff
@@ -2,11 +2,44 @@
 <Wix xmlns="http://schemas.microsoft.com/wix/2006/wi"  xmlns:util="http://schemas.microsoft.com/wix/UtilExtension">
     <Fragment>
         <UI Id="MongoWixUI">
-            <!-- Base our MSI on the FeatureTree based Wix set, and add our custom dialog boxes -->
-            <UIRef Id="WixUI_FeatureTree" />
+            <!-- Based on WixUI_FeatureTree, with our custom dialog boxes -->
+          <UIRef Id="WixUI_Common" />
+          <TextStyle Id="WixUI_Font_Normal" FaceName="Tahoma" Size="8" />
+          <TextStyle Id="WixUI_Font_Bigger" FaceName="Tahoma" Size="12" />
+          <TextStyle Id="WixUI_Font_Title" FaceName="Tahoma" Size="9" Bold="yes" />

```

Comment:
Can you provide some background on why these font selections were added?

Reply:
@nirbar - ping on this comment?

Reply:
@acmorrow They are simply copied from WiXUI_FeatureTree